### ISSUE TITLE
MacOS 10.15.3

### ISSUE THREAD

**Issue by camhart** [bencevans/screenshot-desktop#159]
**Describe the bug**
Screenshot-desktop will fail to capture screenshots on MacOS 10.15.3 **some of the time** (used within an Electron 8 app).  Here is the error that's reported:

`Error: ENOENT: no such file or directory, open '/private/tmp/PKInstallSandbox.JP5MQE/tmp/2020122-2588-jzslvo.ryhpm.jpg'`

What's strange, is this error doesn't occur 100% of the time.  Once it occurs once though, it appears to occur 100% of the time afterwards until the machine restarts (restarting the application doesn't appear to solve it).  It may be happening only after a fresh install.

This could be https://www.npmjs.com/package/temp 's problem more than screenshot-desktop--but figured I'd put the issue here in case others run into it.

**Comment by bencevans** [comment]
I wonder if it's related to the issues we've seen in https://github.com/bencevans/screenshot-desktop/issues/156 regarding requiring Security & Privacy permissions.

**Comment by camhart** [comment]
@bencevans I figured out what's causing it.  I was already granting the Screen Capture permission, however in order for the application to gain that permission it must be restarted.  I was allowing the application to shutdown, but the application would *auto-restart* itself.  That auto-restart fails to qualify it seems.  The user must be the one to launch the application after granting the permission.  Once I did that, the error goes away.
